pyvrp/cpp/Individual_bindings.cpp

```diff
@@ -29,6 +29,15 @@ PYBIND11_MODULE(_Individual, m)
         .def("has_excess_capacity", &Individual::hasExcessCapacity)
         .def("has_time_warp", &Individual::hasTimeWarp)
         .def("__eq__", &Individual::operator==)
+        .def(
```

Comment:
Can we now also remove the copy constructor binding above? (the last of the `py::init`'s).